keyboards/mb44/info.json

```diff
@@ -0,0 +1,202 @@
+{
+    "keyboard_name": "MB-44",
+    "url": "",
+    "maintainer": "melonbred",
+    "width": 3,
+    "height": 2,
+    "layouts": {
+        "LAYOUT_default": {
+            "layout": [
+                {"x":0, "y":0},
+                {"x":1, "y":0},
+                {"x":2, "y":0},
+                {"x":3, "y":0},
+                {"x":4, "y":0},
+                {"x":5, "y":0},
+                {"x":6, "y":0},
+                {"x":7, "y":0},
+                {"x":8, "y":0},
+                {"x":9, "y":0},
+                {"x":10, "y":0},
+                {"x":11, "y":0, "w":1.75},
+                {"x":0, "y":1, "w":1.25},
+                {"x":1.25, "y":1},
+                {"x":2.25, "y":1},
+                {"x":3.25, "y":1},
+                {"x":4.25, "y":1},
+                {"x":5.25, "y":1},
+                {"x":6.25, "y":1},
+                {"x":7.25, "y":1},
+                {"x":8.25, "y":1},
+                {"x":9.25, "y":1},
+                {"x":10.25, "y":1},
+                {"x":11.25, "y":1, "w":1.5},
+                {"x":0, "y":2, "w":1.75},
+                {"x":1.75, "y":2},
+                {"x":2.75, "y":2},
+                {"x":3.75, "y":2},
+                {"x":4.75, "y":2},
+                {"x":5.75, "y":2},
+                {"x":6.75, "y":2},
+                {"x":7.75, "y":2},
+                {"x":8.75, "y":2},
+                {"x":9.75, "y":2},
+                {"x":10.75, "y":2},
+                {"x":11.75, "y":2},
+                {"x":0, "y":3},
+                {"x":1, "y":3},
+                {"x":2, "y":3},
+                {"x":3, "y":3, "w":6.25},
+                {"x":9.75, "y":3},
+                {"x":10.75, "y":3},
+                {"x":11.75, "y":3}
+            ]
+        },
+        "2u_split": {
```

Comment:
etc.

Reply:
`LAYOUT_` added to all layouts. Sorry about that!